tests/testthat/helper-funs.R

```diff
@@ -0,0 +1,35 @@
+basicScatP <- function() {
+  ggplot2::ggplot(data = mtcars) +
+    ggplot2::geom_point(ggplot2::aes(x = wt, y = drat)) 
+}
+
+ggMarg2 <- function(type, ...) 
+  ggMarginal(p = basicScatP(), type = type, ...)
+
+funList <- 
+  list(
```

Comment:
Let me show you what I have planned for testing under multiple versions of ggplot2 - There are a few small pieces of render-figs.R that need to be changed, but I think you will find that it is set up with the future in mind.